ideas/151-ens-usernames.md

```diff
@@ -0,0 +1,66 @@
+# Username ENS subdomain Registrar
+
+## Preamble
+
+    Idea: 151-ens-usernames
+    Title: Username ENS subdomain Registrar
+    Status: Draft
+    Created: 2017-11-20
+    Replaces: 027-ens-usernames
+
+## Summary
+
+Eliminates the need to copy/scan - and worse, type - long hexadecimal addresses / public keys, by providing an [ENS](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-137.md) subdomain registry and recognition of ENS names in Status for interacting with other people in Status.
+
+## Swarm Participants
+
+- Lead Contributor: Ricardo Guilherme Schmidt
+- Contributor: Richard Ramos
+- UX (Mist): Alexandre Van de Sande 
+- UX (Status): Hester Bruikman
+
+## Product Overview
+
+A bad experience in most cryptosystems, aswell Status, is the requirement of typing or scanning the public keys and addresses to interact with other users in the network.
+ENS disrupt centralized DNS: like DNS resolves into IPs, ENS resolve into ethereum addresses (and more) trustlessly.
+Status would introduce a option for users registering a human readable contact username and its URI would be used in place of QR codes / private keys.
+A small fee is required, so users are buying the subdomain, and therefore could be exchanged with other users, this network effect is ignored by this idea.
+
+### Product Description
+
+Users would be able to:
+- buy a subdomain from one of the ENS domains owned by Status.
+- find users by using the configured ENS Resolver for EMS address (subdomain.domain.eth)
+- search/suggest domain for a username.
+- send funds by typing their ENS address.
+
+Subdomain owners would be able to:
+- change the owner of the subdomain
+- change the resolver of the subdomain
+
+Democracy (or authority) would be able to:
+- add new domains available with its own price
+- change domains price
+- change owner of domain (required for update of subdomian registry)
+
+### Minimum Viable Product
```

Comment:
Can we break this down a bit? What are the blockers for each thing?

- What's required for registering?
- "Users can add friends" - do you mean in Status app or in contract?
- "Basic UI for call" - what does for call mean? Maybe we can separate out the mockups from implementing in core client?